app/navigation/OtherOfficialsNavigator.scala

```diff
@@ -190,22 +195,32 @@ class OtherOfficialsNavigator @Inject()(implicit frontendAppConfig: FrontendAppC
     }
 
     case IsOtherOfficialNinoPage(index) => userAnswers: UserAnswers => userAnswers.get(IsOtherOfficialNinoPage(index)) match {
-      case Some(_) => routes.DeadEndController.onPageLoad() // TODO summary page
-      case _ =>  routes.SessionExpiredController.onPageLoad()
+      case Some(true) if userAnswers.get(OtherOfficialsNinoPage(index)).isDefined => redirectToPlaybackPage(index)
+      case Some(true) => otherOfficialRoutes.OtherOfficialsNinoController.onPageLoad(PlaybackMode,index)
+      case Some(false) if userAnswers.get(OtherOfficialsPassportPage(index)).isDefined =>
+        redirectToPlaybackPage(index)
+      case Some(false) => otherOfficialRoutes.OtherOfficialsPassportController.onPageLoad(PlaybackMode,index)
+      case _ => routes.SessionExpiredController.onPageLoad()
     }
 
     case OtherOfficialsNinoPage(index) => userAnswers: UserAnswers => userAnswers.get(OtherOfficialsNinoPage(index)) match {
       case Some(_) => redirectToPlaybackPage(index)
       case _ =>  routes.SessionExpiredController.onPageLoad()
     }
 
+    case OtherOfficialsPassportPage(index) => userAnswers: UserAnswers => userAnswers.get(OtherOfficialsNinoPage(index)) match {
+      case Some(_) => addressLookupRoutes.OtherOfficialsAddressLookupController.initializeJourney(index, PlaybackMode )
+      case _ =>  redirectToPlaybackPage(index)
+    }
+
```

Comment:
this means the user would be sent to the address lookup page when they change their passport details. is this really expected behaviour?